### PR TITLE
Add basic index.html page

### PR DESCRIPTION
**Pull request by mezarin**

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            <h2>Open Liberty</h2>
+            <p>
+                The <a href="http://openliberty.io/">Open Liberty project</a> is an open
+                community supporting Java EE and <a href="http://microprofile.io/">MicroProfile</a>, a runtime starting fast and made for running cloud-native microservices.
+            </p>
+        </div>
+        <div>
+            <h2>Odo</h2>
+            <p>
+                The <a href="http://odo.dev/">odo CLI tool </a> is, iterative, and straightforward way for developers who write, build, and deploy applications on Kubernetes and OpenShift.
```
Comment (scottkurz) [inline src/main/webapp/index.html]: is, iterative, and straightforward way  =>
is an iterative and straightforward way

who write, build, and deploy =?
to write, build, and deploy

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        <h1>Welcome To Your Open Liberty Microservice</h1>
+        <div>
+            <h1>Further Reading</h1>
+            <p>Please see below for information on related technologies.</p>
+        </div>
+        <div>
+            <h2>Open Liberty</h2>
+            <p>
+                The <a href="http://openliberty.io/">Open Liberty project</a> is an open
+                community supporting Java EE and <a href="http://microprofile.io/">MicroProfile</a>, a runtime starting fast and made for running cloud-native microservices.
```
Comment (scottkurz) [inline src/main/webapp/index.html]: Java EE =>
<a href="Jakarta EE https://jakarta.ee/>Jakarta EE</a>

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            </p>
+        </div>
+        <div>
+            <h2>Odo</h2>
+            <p>
+                The <a href="http://odo.dev/">odo CLI tool </a> is, iterative, and straightforward way for developers who write, build, and deploy applications on Kubernetes and OpenShift.
+                The odo tool abstracts away complex Kubernetes and OpenShift concepts for the developer.
+            </p>
+            <p>
+                One of Odo's core concepts is a the concept of a <a href="https://docs.devfile.io/devfile/2.0.0/user-guide/index.html">devfile</a>.
```
Comment (scottkurz) [inline src/main/webapp/index.html]: One of Odo's core concepts is a the concept =>
One of Odo's core concepts is the concept

**Review by scottkurz — CHANGES_REQUESTED** [review]

File: src/main/webapp/index.html (hunk 1 above)
Comment (mezarin) [reply src/main/webapp/index.html]: Thanks Scott. I updated it to reflect this and the reset of your comments.

**Review by scottkurz — APPROVED** [review]

**Pull request merged**